content/docs/command-reference/move.md

```diff
@@ -30,6 +30,10 @@ If the destination path (`dst`) already exists and is a directory, the source
 code file or directory (`src`) is moved unchanged into this folder along with
 the corresponding `.dvc` file.
 
+`dvc move` doesn't support renaming outputs of pipeline stages (`dvc run`), it
+only renames data source files and directories (added using `dvc add` and
+`dvc import`). The pipeline outputs have to be renamed manually.
```

Comment:
Some of the changes you made to Dmitry's text here didn't improve it:

- He started with "data source files and directories" (generic concept). You start with "outputs" (DVC jargon). What is that? New users won't know.
  > We have a glossary entry for that term BTW, so please use `<abbr>outputs</abbr>` for the first instance.

- He uses "outputs of pipelines", you appended "stages" to the phrase, which is redundant. All pipelines have stages. Most of the times we only need to mention one or the other. I would actually go for "stage outputs (**see** `dvc run`)" (I reserve the term "pipeline" for actual documents about pipelines).

- You correctly identified the phrase "creaded by `dvc add` and `dvc import`" and improved it to "added using `dvc add` and `dvc import`). But it's still not perfect: "Added using add"? Sounds strange. Also, is that parenthesis needed?

- Does the last sentence need "The"? We're not talking about any specific pipeline outputs (I'd also change that to "Stage outputs BTW".

Reply:
Anyway, I'd rather not rewrite every sentence but for now during bonding I'll be giving you these specific tips as writing training. I will leave more general feedback progressively 🙂

Reply:
All that said... Really this information is super important and should be part of the very first paragraph. Please rewrite the first paragraphs or 2 to make extra clear that `move` is meant for .dvc files only. You don't have to keep the existing text, it can be totally different, as long as it's correct, complete, and as brief as possible. Thanks

Reply:
Thanks for grammar inputs. Updated the doc.